scripts/generate-api.sh

```diff
@@ -11,13 +11,13 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 cd "$SCRIPT_DIR/.."
```

Comment:
Generate script je teď generický (`API_NAME`). Taky spadne, pokud se nepodaří přesun souborů openapi

Reply:
To měníš záměrně ty file permissions u tohodle souboru?

Reply:
Teď nevím, co myslíš tím file permissions. Sám u sebe dávám `sudo chown -R $UID .` abych si fixnul to, že mi nepovoluje upravovat vygenerovaný soubory nebo soubory, který prošly přes `make fix`. To se nějak propaguje přes git?

Reply:
Tohle, odstranil jsi executable permission.
![CleanShot 2022-11-02 at 15 50 29](https://user-images.githubusercontent.com/215660/199521958-f57b8bd5-cda3-49da-a4f0-80fdc5dc280a.jpg)

Reply:
Aha, jak to vrátím zpět?